### ISSUE TITLE
Windows support?

### ISSUE THREAD

**Issue by NevarroGuildsman** [jkirkcaldy/plex-utills#11]
I'm interested in getting this working on my Plex server which is running on Windows Server 2019. I've installed Python including pip and installed the dependencies. My media is on a mapped drive - will putting the Z:\Media path in the config file work? Or are there things in the scripts that would need to be modified as well?

**Comment by jkirkcaldy** [comment]
In theory that should work. But I haven't tested on windows yet.

**Comment by Shayne55434** [comment]
> 
> 
> I'm interested in getting this working on my Plex server which is running on Windows Server 2019. I've installed Python including pip and installed the dependencies. My media is on a mapped drive - will putting the Z:\Media path in the config file work? Or are there things in the scripts that would need to be modified as well?

Were you able to get this working? I've tried running it remotely and locally and I can't seem to figure it out. My files are located here:
D:\Media\4K Movies

What values would I use for mountedpath and plexpath? I've tried all sorts of combinations and keep getting errors like so:
![image](https://user-images.githubusercontent.com/37595910/121048735-ad29c500-c77c-11eb-95c9-063a44baf58e.png)

**Comment by NevarroGuildsman** [comment]
Yep, looks like I get the same thing for the 4k_hdr_poster.py script.
![image](https://user-images.githubusercontent.com/41127216/123013119-f5123400-d399-11eb-856c-f7a1407ab61f.png)

The 4klibrary.py script ran fine and did add the correct tags to the files.

**Comment by JordanBerger** [comment]
In `config.ini`, add 2 double-backslashes to both PLEXPATH and MOUNTEDPATH (these should be the same and can be any drive that contains your content. For example my config looks like:
```
PLEXPATH = F:\\Movies
MOUNTEDPATH = F:\\Movies
```

**Comment by jkirkcaldy** [comment]
I have tested this and it is now resolved in the latest commit. 

If you have your Plex share mounted to your Z: drive then you would set your mounted path as `Z:` If you need to specify a folder inside your mounted share, use a forward slash, not a backward slash. 

The only time you would need to add a slash is if you need to open a folder inside your plex share. So if you need to click on your network share and then open a folder called `plex_media` then your path should be `Z:/plex_media`

If you don't need to open any folders on your shared drive then you should leave it as the drive letter only. 

I was getting errors with the escaped characters so for the comment above, you will likely have better luck using this in the config:
```
PLEXPATH = F:/Movies
MOUNTEDPATH = F:/Movies
```
Python will still be able to access the files there. 

This is only needed if you enable the poster backup.
